hedera-mirror-rest/__tests__/service/fileDataService.test.js

```diff
@@ -37,25 +37,25 @@ const files = [
   {
     consensus_timestamp: 1,
     entity_id: 112,
-    file_data: '0a1008b0ea0110cac1181a0608a0a1d09306121008b0ea0110e18e191a0608b0bdd09306',
+    file_data: Buffer.from('0a1008b0ea0110cac1181a0608a0a1d09306121008b0ea0110e18e191a0608b0bdd09306', 'hex'),
```

Comment:
nit: use a shared method to 
```
const decode = (contents) => {
  return Buffer.from(contents, 'hex');
}
```

then

```suggestion
    file_data: decode('0a1008b0ea0110cac1181a0608a0a1d09306121008b0ea0110e18e191a0608b0bdd09306'),
```